payments/mobile-pay/other-features.md

```diff
@@ -533,7 +533,7 @@ authorization amount.
 
 ```mermaid
 sequenceDiagram
-  Merchant->>PayEx: POST <creditcard capture>
+  Merchant->>PayEx: POST <mobilepay capture>
```

Comment:
```suggestion
  Merchant->>PayEx: POST <MobilePay Online capture>
```

Reply:
Since the `capture` applies directly to the API, which is called `mobilepay`, I don't agree with this change.